homeassistant/components/vallox/__init__.py

```diff
@@ -164,7 +162,7 @@ def get_profile(self):
             self._profile = await self._client.get_profile()
             self._valid = True
 
-        except OSError as err:
+        except (OSError, InvalidMessage) as err:
```

Comment:
Please catch interface specific exceptions in the library. :+1:

Reply:
Sure, makes sense. Keeping the changes that delay the first update to reduce probability of the error surfacing in the websockets library.